migrations/2_deploy_xcltoken.js

```diff
@@ -38,8 +40,13 @@ module.exports = function(deployer) {
     	console.log('XcelToken.address :' + XcelToken.address);
     	return xcelToken.balanceOf(StepVesting.address);
 	}).then((balance) => {
-		console.log('StepVesting.address balance :' + balance);
+		  console.log('StepVesting.address balance :' + balance);
     	return xcelToken.initiateTeamVesting(StepVesting.address);
+   }).then( () => {
+      console.log('beneficiary for OneTimeTokenVesting : ' + beneficiary2nd);
+      return deployer.deploy(OneTimeTokenVesting, beneficiary2nd, start, 360, true);
+   }).then( () => {
+      console.log("OneTimeTokenVesting address : " + OneTimeTokenVesting.address);
```

Comment:
lets transfer 100 tokens from Xceltoken using transfer - makes it easy to test from ganache console

Reply:
done